fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py

```diff
@@ -1046,6 +1075,13 @@ def encode_list(
                     f"both padding token and padding id are None, but padding length is {max_len}. It's possible that it was set for truncation alone."
                 )
 
+        # Check if we have any unknown tokens in our input
+        if self.count_unknowns(merged_encoding) > 0:
+            if on_unknown == "raise":
+                raise Exception("Encountered unknown tokens in the input")
+            elif on_unknown == "warn":
+                warn("Encountered unknown tokens in the input")
```

Comment:
please add another "else" and raise Exception if on_unknown got an unexpected value (and print what the user provided)
Additionally, consider adding 'ignore' option.

Reply:
done